Classic_RPG/src/entities/character/Warrior.java

```diff
@@ -26,7 +28,9 @@ public void specialHit(Entity target) {
 	}
 	
 	public void receiveAttack(int dmgIncoming) {
-		if(defense_position) {
+		int success = getCharacteristics().getStrength() + 30;
+		TestOutput result1D100 = Random.test1D100(success);
+		if(defense_position && result1D100 != TestOutput.CRITICAL_FAILURE && result1D100 != TestOutput.FAILURE) {
```

Comment:
Not sure != got the priority over &&, add "()" around "result1D100 != TestOutput...."
With this code a CRITICAL_FAILURE has same consequences as FAILURE

CRITICAL_SUCCESS has same consequences as SUCCESS